homework-2/src/templates/header.html

```diff
@@ -0,0 +1,31 @@
+<div class="header">
```

Comment:
use header tag. Have yu heard about HTML5?

Reply:
div tag it's a bad habit, header and footer Done